data-sources/mssql-customqueries/config.yml

```diff
@@ -0,0 +1,9 @@
+id: mssql-customqueries
+displayName: MSSQL On-Host Integration w/ Custom Queries
+description: |
+  Display MSSQL information collected from custom queries
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration/
+icon: logo.svg
```

Comment:
@csalvador-nr Sorry, this isn't what I meant. What you've added here is a `data source`, and there's an existing one for nri-mssql already. What you need to do is add `quickstarts/msqql-customqueries/config.yml` (and logo file) and add a quickstart definition that references the mssql data source ID [like so](https://github.com/newrelic/newrelic-quickstarts/blob/release/quickstarts/mssql/config.yml#L25-L26).

Reply:
For ref, by using this existing data source, clicking `Install` from the public site will allow the user to get to the in-product install experience for MSSQL:
![image](https://github.com/newrelic/newrelic-quickstarts/assets/6722433/5f7695dc-c6d1-4c6f-8c55-c76268043c96)

Reply:
My bad, I believe commit c8b0bbf fix it.